client-code-gen/gen/model/app-env-type.ts

```diff
@@ -0,0 +1,32 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Forest Access Management - FAM - API
+ *  Forest Access Management API used by the Forest Access Management application to Define who has access to what apps, and what roles they will operate under  once access is granted. 
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: guy.lafleur@montreal.canadians.ca
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+
+/**
+ * An enumeration.
+ * @export
+ * @enum {string}
+ */
+
+export const AppEnvType = {
```

Comment:
I think this file is unnecessary, replaced by app-env.ts, but I could be wrong. Maybe try deleting the gen directory before regenerating the files to avoid leaving unnecessary files?

Reply:
Thanks Basil! This one is removed.